src/eigen/lobpcg_hyper_impl.jl

```diff
@@ -213,9 +213,6 @@ function ortho!(X, Y, BY; tol=2eps(real(eltype(X))))
     end
     vprintln("ortho choleskys: ", ninners) # get how many Choleskys are performed
 
-    # @assert (norm(BY'X)) < tol
```

Comment:
These are useful postconditions, put a comment instead if you want to remove them